logic/block_reward.js

```diff
@@ -33,13 +33,13 @@ const __private = {};
 class BlockReward {
 	constructor() {
 		// Array of milestones
-		this.milestones = constants.REWARDS.MILESTONES;
+		this.milestones = REWARDS.MILESTONES;
```

Comment:
Why do we need to assign those instead of using directly?

Reply:
Removed: `this.milestones`.